torch/_subclasses/fake_impls.py

```diff
@@ -595,6 +596,255 @@ def convert(t, mem_fmt):
             )
 
 
+@register_op_impl(aten._scaled_dot_product_flash_attention.default)
+def meta__scaled_dot_product_flash(fake_mode, func, *args, **kwargs):
+    _, kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    query = kwargs["query"]
+    key = kwargs["key"]
+    return_debug_mask = kwargs["return_debug_mask"]
+    # unused: value, dropout_p, is_causal, scale
+
+    def convert_tensor(t, device):
+        return FakeTensor(fake_mode, t, device)
+
+    batch_size = query.size(0)
+    num_heads = query.size(1)
+    max_seqlen_batch_q = query.size(2)
+    head_dim = query.size(3)
+    max_seqlen_batch_k = key.size(2)
+
+    query_t = query.transpose(1, 2)
+    attention = torch.empty_like(query_t).transpose(1, 2)
```

Comment:
dumb q, more just pattern matching why doesn't this need `convert_tensor`?

Reply:
I think empty_like returns a fake tensor already, so it doesn't need to be wrapped (not really sure how this works, I just noticed it throws errors if I try to wrap it)

Reply:
ohh duh  cause query_t is fake